### ISSUE TITLE
bug: ios 15, safari bottom url bar does not expand when tapping multiple inputs

### ISSUE THREAD

**Issue by chrisventura** [ionic-team/ionic-framework#23994]
### Prequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

- [ ] v4.x
- [X] v5.x
- [ ] v6.x

### Current Behavior

On iOS 15 Safari, after dismissing a keyboard for an input with autocomplete, the view doesn't get reset properly to the proper height resulting in a big space at the bottom of the app that remains until refresh.

Notice that after dismissing the keyboard from a regular input, the height resets fine. Bug seems to only occur after dismissing a keyboard with autocomplete.

https://user-images.githubusercontent.com/4053234/135306987-94ef4052-8430-42f6-aa7a-167044d52c8d.MP4

### Expected Behavior

I expect the app to fill the height of the screen after dismissing the keyboard.

### Steps to Reproduce

1. Create an ionic angular app 
    - I used `ionic start photo-gallery tabs --type=angular --capacitor`
2. Replace content of `tab.page.html` with something like.. 
```
<ion-content>
  <h1>this is content</h1>
  <h1>this is content</h1>
  <h1>this is content</h1>
  <h1>this is content</h1>
  <h1>this is content</h1>
  <h1>this is content</h1>
  <h1>this is content</h1>
  <h1>this is content</h1>
  <h1>this is content</h1>
  <h1>this is content</h1>
  regular input
  <br /><input />
  <br />
  <br />
  telephone input with autocomplete and type
  <br />
  <input name="phone-number" type="tel" autocomplete="tel"/>
  <h1>this is content</h1>
  <h1>this is content</h1>
  <h1>this is content</h1>
  <h1>this is content</h1>
  <h1>this is content</h1>
  <h1>this is content</h1>
  <h1>this is content</h1>
  <h1>this is content</h1>
  <h1>this is content</h1>
  <h1>this is content</h1>
  <h1>this is content</h1>
  <h1>this is content</h1>
  <h1>this is content</h1>
</ion-content>
<ion-footer>
  <ion-toolbar>
    <ion-button>footer button</ion-button>
  </ion-toolbar>
</ion-footer>
```
3. Run application 
  - I use `ng serve --host 0.0.0.0` so I can open it on my iPhone.
4. Open with Safari on iOS 15 (with tab bar on the bottom)

### Code Reproduction URL

https://github.com/chrisventura/photo-gallery

### Ionic Info

```
Ionic:

   ionic (Ionic CLI)             : 4.8.0 (C:\Users\Chris.Ventura\AppData\Roaming\npm\node_modules\ionic)
   Ionic Framework               : @ionic/angular 5.8.0
   @angular-devkit/build-angular : 12.1.4
   @angular-devkit/schematics    : 12.2.6
   @angular/cli                  : 12.1.4
   @ionic/angular-toolkit        : 4.0.0

Capacitor:

   capacitor (Capacitor CLI) : 3.2.3
   @capacitor/core           : 3.2.3

System:

   NodeJS : v16.9.1 (C:\Program Files\nodejs\node.exe)
   npm    : 7.24.0
   OS     : Windows 10
```

**Comment by liamdebeasi** [comment]
Thanks for the issue. We are tracking this in https://github.com/ionic-team/ionic-framework/issues/23423. (See "Investigate how Safari's new floating tab bar impacts Ionic applications")

In particular, it looks like we need to change some container element to have `height: 100vh` instead of `height: 100%`.

Please follow the #23423 for updates. Thank you!

**Comment by chrisventura** [comment]
I've attempted to add `height: 100vh` to `html` in my projects, but that had some other effects to behaviour, such as.. causing Safari UI to automatically collapse on scroll, and I wasn't sure if that was going against Ionic's intended behaviour.

**Comment by liamdebeasi** [comment]
Safari's UI should automatically collapsed on scroll in Ionic apps. Right now it does not because we do not have `height: 100vh`.

**Comment by liamdebeasi** [comment]
Are there any other side effects you noticed? I am looking to tackle this soon in #23423 so any other insights you have would be really helpful.

**Comment by chrisventura** [comment]
I've also noticed some interesting things with overscrolling and iOS as well. But I'm not exactly sure what the problem is here. 

When using `height: 100vh; !important` on the HTML tag, scrolling becomes a little bit buggy. ie. the footer jumps around when the Safari UI collapses and the vh changes (not smooth), and scrolling can occasionally get stuck from time to time when scrolling all the way to the top of a scrollable div. Feels kind of like an iOS bug, but I'm not too sure. 

I added a branch to https://github.com/chrisventura/photo-gallery called `scrolling-bug` if you would like to try it out.
